src/components/CharactersList.js

```diff
@@ -1,8 +1,86 @@
-import Characters from "./Characters";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import Filter from "./Filter";
+import "../CharacterList.css";
+
 function CharactersList() {
```

Comment:
🔬https://github.com/eduardoccmelo/react-rick-and-morty/pull/1#discussion_r611608924